lib/guard/coffeelint.rb

```diff
@@ -8,7 +8,7 @@ class Coffeelint < Plugin
 
     def initialize(options = {})
       super
-      @config_file = options[:config_file]
+      @config_file = options[:config_file] || 'config/coffeelint.json'
```

Comment:
Yes, I didn't think of that. Possibly a better solution would be to tell users in the README that your gem defaults to `./coffeelint.json` :-)